yarn.lock

```diff
@@ -3572,6 +3572,11 @@ dom-accessibility-api@^0.5.1:
   resolved "https://registry.yarnpkg.com/dom-accessibility-api/-/dom-accessibility-api-0.5.16.tgz#5a7429e6066eb3664d911e33fb0e45de8eb08453"
   integrity sha512-X7BJ2yElsnOJ30pZF4uIIDfBEVgF4XEBxL9Bxhy6dnrm5hkzqmsWHGTiHqRiITNhMyFLyAiWndIJP7Z1NTteDg==
 
+dom-mutator@^0.6.0:
```

Comment:
Is yarn.lock change needed?

Reply:
it is a dependency, not sure why it is not in yarn.lock from the start?

Reply:
Since it's not expected for experiment-tag to be used directly